src/ildasm/dasm.cpp

```diff
@@ -7873,12 +7877,13 @@ BOOL DumpFile()
                     {   // initialized data
                         szptr+=sprintf_s(szptr,SZSTRING_REMAINING_SIZE(szptr),"%s%8.8X = %s (",szTls,fromPtr,KEYWORD("bytearray"));
                         printLine(g_pFile,szString);
-                        sprintf_s(szString,SZSTRING_REMAINING_SIZE(szptr),"%s                ",g_szAsmCodeIndent);
+                        szptr = szString;
+                        sprintf_s(szptr,SZSTRING_REMAINING_SIZE(szptr),"%s                ",g_szAsmCodeIndent);
```

Comment:
We haven't flushed the szString that has this indention yet
I think it is flushed out in ```DumpHexbytes``` including blob data in it.
In particular, inside the function, we compute ```szPtr``` from ```szString``` again after the this indention and then append the blob data.
At each line (k=16), we flush the beginning of ```szString``` which is a parameter not in the global one, and reset the line with indention. In fact, in this use, I think they are effectively identical so we might not even need to pass such parameter. 

Well... I hate this kind of operation in this code, and I'm not sure whether I'm reading the code correctly. Since you're here, can you validate the indention is expected? In either case, this won't impact the correctness other than aesthetic thing in display.